src/chttpd/src/chttpd_changes.erl

```diff
@@ -212,8 +231,10 @@ filter(Db, Change, {doc_ids, Style, DocIds}) ->
     end;
 filter(Db, Change, {selector, Style, {Selector, _Fields}}) ->
     Docs = open_revs(Db, Change, Style),
-    Passes = [mango_selector:match(Selector, couch_doc:to_json_obj(Doc, []))
-        || Doc <- Docs],
+    Passes = [
+        mango_selector:match(Selector, couch_doc:to_json_obj(Doc, []))
+     || Doc <- Docs
```

Comment:
0. I would indent `||` on the same level as previous line. But it reads good in either case.

Reply:
+1